scapy/layers/inet6.py

```diff
@@ -3966,6 +4007,7 @@ def _load_dict(d):
         d[k] = _get_cls(v)
 
 
+_load_dict(icmp6rplcodes)
```

Comment:
This will never be called: the contrib module will always be called after `inet6`, meaning this will always be empty.

Reply:
Fixed: removed icmp6rplcodes and some other relevant dicts.